src/Gitlab.purs

```diff
@@ -61,11 +147,13 @@ getProjects (BaseUrl baseUrl) (Token token) = do
       throwError $ error ("Failed to parse projects: " <> show e)
     Right projects -> pure projects
 
-getJobs :: forall a. BaseUrl -> Token -> Project -> Aff (ajax :: AJAX | a) Jobs
+getJobs :: forall a.
+           BaseUrl -> Token -> Project
+           -> Aff (ajax :: AJAX | a) Jobs
 getJobs (BaseUrl baseUrl) (Token token) project = do
   let url = baseUrl
             <> "/api/v4/projects/"
-            <> show project.id
+            <> writeJSON project.id -- same as show in this case
```

Comment:
What makes it better than `show` though? You know that I'm not a fan of show because of how much it's overloaded and lawless, but I think it's OK for primitive types (like `Int` in that case), it would read straightforwardly and won't work  if that `id` becomes something more complex (like a record).